action.yml

```
@@ -0,0 +1,14 @@
+# action.yml
+name: 'Check DCO Signoff'
+description: 'A reusable action that checks if all commit has a DCO signoff'
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.12
+
+    - name: Run Python script
```

Since this is printed in CI, this should be a bit more specific:

```suggestion
    - name: Check DCO
```

(like in https://github.com/christophebedard/dco-check/blob/master/.github/workflows/dco.yml)